ci/azure/linux_script

```diff
@@ -59,6 +59,10 @@ unset CXX
 
 make $JOBS install
 
+# look for formatting errors
+release/bin/zig fmt --check ../lib/std ../doc ../src ../tools ../test ../build.zig
```

Comment:
Suggestion, to make it more clear what to do for a contributor whose PR is not passing:

` || echo "Please run 'zig fmt' to fix the non-conforming files listed above."`